iambic/plugins/v0_1_0/aws/iambic_plugin.py

```diff
@@ -76,6 +76,14 @@ class AWSConfig(ConfigMixin, BaseModel):
     accounts: list[AWSAccount] = Field(
         [], description="A list of AWS Accounts to be managed by iambic"
     )
+    enable_iam_user_credentials: Optional[bool] = Field(
+        default=True,
+        description="If true, IAM User templates will include "
+        "summary info of the user password and access keys. "
+        "It will also allow iambic to delete a user passwords and disable access keys. "
+        "Enable is not currently supported until "
```

Comment:
For docs clarity, probably rephrase "Enable is not currently supported" to "Enabling IAM user access keys from iambic"